lib/testmap.py

```diff
@@ -178,6 +178,18 @@
         '_manual': [
         ]
     },
+    'ptoscano/subscription-manager-cockpit': {
+        'main': [
+        ],
+        '_manual': [
+            'rhel-8-5',
+            'rhel-8-6',
+            'rhel-8-6/subscription-manager-1.28.29',
```

Comment:
Are you sure these are scenarios, not branch names?

Reply:
Yes, as my idea is to use the scenario feature to test the `main` branch of sub-man-cockpit with multiple branches of sub-man, to ensure it is compatible.

Reply:
On top of that, a longer term idea of mine is to add a "system" scenario to use system packages rather than a git version of sub-man.